NumericInput/NumericInput.js

```diff
@@ -158,7 +177,7 @@ export default class NumericInput extends Component {
         const iconStyle = [style.icon, this.props.iconStyle]
         const totalWidth = this.props.totalWidth
         const totalHeight = this.props.totalHeight ? this.props.totalHeight : (totalWidth * 0.4)
-        const inputWidth = this.props.type === 'up-down' ? (totalWidth * 0.6) : (totalWidth * 0.4)
+        const inputWidth = this.props.inputWidth? this.props.inputWidth: this.props.type === 'up-down' ? (totalWidth * 0.6) : (totalWidth * 0.4)
```

Comment:
Please add space between this.props.inputWidth and ?  and also between this.props.inputWidth and :